pkgs/os-specific/linux/wireguard/wg-default.sh

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env bash
```

Comment:
This has code from wg-quick and therefore needs my copyright readded.

```
# Copyright (C) 2016-2017 Jason A. Donenfeld <Jason@zx2c4.com>. All Rights Reserved.
#
# This file is licensed under the GPLv2 as published by the Free Software Foundation.
```

Reply:
Somebody must have already told you that "All Rights Reserved" contradicts the rights granted by GPLv2… There is an [anecdote](https://en.wikipedia.org/w/index.php?title=Copyleft&oldid=809250700#cite_ref-autogenerated1_21-0) about that.

Reply:
```
zx2c4@thinkpad ~/Projects/linux $ grep -ri 'all rights reserved' | wc -l
8372
```

Have your lawyers talk to the Linux Foundation's lawyers about that then. I'm more likely to go with whatever decision lawyers make than with what a random guy on a github issue claims.

Until that happens, you can respect my copyright line, or not use my code. I won't waste my time arguing fake legal nonsense on a github issue.